contracts/multisig/src/migrations/v_0_4.rs

```diff
@@ -0,0 +1,79 @@
+use crate::state::{SIGNING_SESSIONS, VERIFIER_SETS};
+use cosmwasm_std::{DepsMut, Order, Response};
+
+pub fn migrate_verifier_set_ids(
+    deps: &mut DepsMut,
+) -> Result<Response, axelar_wasm_std::ContractError> {
+    let all: Vec<_> = VERIFIER_SETS
+        .range(deps.storage, None, None, Order::Ascending)
+        .collect::<Result<Vec<_>, _>>()?;
+
+    for v in all {
+        VERIFIER_SETS.remove(deps.storage, &v.0);
+        VERIFIER_SETS.save(deps.storage, &v.1.id(), &v.1)?;
+    }
+
+    Ok(Response::default())
+}
+
+pub fn migrate_signing_sessions(
+    deps: &mut DepsMut,
+) -> Result<Response, axelar_wasm_std::ContractError> {
+    SIGNING_SESSIONS.clear(deps.storage);
+
+    Ok(Response::default())
+}
+
+pub fn migrate(deps: &mut DepsMut) -> Result<Response, axelar_wasm_std::ContractError> {
+    // signing sessions should be migrated first, so that way the old ids still point to the verifier sets
```

Comment:
I don't understand this point. You're removing the sessions, so what does it matter where the id points?

Reply:
This was a comment from before, from when the sessions weren't being removed. Can remove the comment now